### ISSUE TITLE
[HOLD Expensify #233363] Desktop - VBA - Console error appears in personal info page

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#13992]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Open app and login with gmail account
2. Go to workspace > Connect bank account
3. Select option manually
4. Continue up to personal info page
5. Fill up and click Save & continue

## Expected Result:
There should be no console error message

## Actual Result:
"uncaugt syntax" error message appears

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?

- [x] MacOS / Desktop

**Version Number:** 1.2.48.2

**Reproducible in staging?:** Yes

**Reproducible in production?:** n/a

**Email or phone of affected tester (no customers):**

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation

![Screen Shot 2023-01-04 at 1 34 38 PM](https://user-images.githubusercontent.com/93399543/210648177-3fc0a107-aaa0-417b-9a5f-9d47c5b3e4e5.png)

**Expensify/Expensify Issue URL:**

**Issue reported by:** Applause - Internal Team

**Slack conversation:**

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by mountiny** [comment]
This feels like it might not be a bug we can do anything about. The console error is most likely coming from Onfido

**Comment by twisterdotcom** [comment]
@kbecciv I confirmed this is for sure coming from Onfido when I recreated it on Desktop too:

<img width="1233" alt="image" src="https://user-images.githubusercontent.com/9133401/211376826-c3648a2b-6e64-4077-945e-0e9ee06f0298.png">

As we did here: https://github.com/Expensify/App/issues/13262#issuecomment-1342165906, could you create an issue for Onfido?

Similar to the comment from @JmillsExpensify here, I'm going to put this on HOLD for Onfido as well. Once you've pasted the issue in onfido-sdk-ui, let me know.

**Comment by mvtglobally** [comment]
@twisterdotcom We do not have a different way to create Onfido issues. @JmillsExpensify Can you help out with this request?

**Comment by twisterdotcom** [comment]
The comment I linked is this:

<img width="1006" alt="image" src="https://user-images.githubusercontent.com/9133401/211920718-a284e608-1df0-482f-9881-bfc333311c6b.png">

If Onfido is delivering us a console error when their SDK is loading, we should fix it upstream in their SDK, if we care about fixing it because it's not something we can fix without pushing it to their SDK - I think at least. That is what we recommended before in the comment I was referencing.

As we likely don't care though, we could skip that entirely and just leave it held on the Onfido API changes anyway. What do you think @JmillsExpensify?

**Comment by mountiny** [comment]
If the onfido flow works, which I assume it does otherwise Applause would caught it, then we could do just post this in the Onfido repo https://github.com/onfido/onfido-sdk-ui but it does not seem they would be very active with addressing those.

Then we can link it here and close this issue.

**Comment by JmillsExpensify** [comment]
I like that plan of action (and thanks @twisterdotcom for the clarifications!).

**Comment by mountiny** [comment]
I can repost it to the onfido repository tomorrow (tan out of time today) unless Ted will do it today

**Comment by twisterdotcom** [comment]
@mountiny - could you help me with their template? Added **No idea** below where I'm unsure.

---

### What was the expected behaviour?

No console errors when loading the SDK.

### What happened instead?

A repeated console error

```
Uncaught SyntaxError: Unexpected token '<' (at ?ra=Hu8E2MnjUd9d&alias=onfido-js-sdk.com&instance=onfidojssdkwoopra&ka=24000&meta=&screen=1800x1169&language=en-GB&app=js-client&referer=&idle=0&vs=r&cookie=CYnWGGTyVF6y:1:1)
```

<img width="1233" alt="image" src="https://user-images.githubusercontent.com/9133401/211371570-75412f17-18e1-40fb-9d06-eab7c54ce13b.png">

### Version info:

- Onfido SDK: 8.3.0
- OS: macOS
- Browser: https://www.electronjs.org/
- Device: MacBook Pro 14", 2021
- Node: **No idea**
- Framework or relevant dependencies (if applicable): **No idea**

### Integration configuration:

Please add code snippet(s) of your integration with the Onfido SDK and all customisations of it.

**No idea**

### Steps to reproduce:

1. Download new.expensify.com from https://new.expensify.com/NewExpensify.dmg
2. Follow the instructions here

Please describe in as much detail as possible how to reproduce the problem

### Reproducible demo:

Please provide a replication of the bug in our latest or LTS version as a demo. We have a basic [jsfiddle](https://jsfiddle.net/gh/get/library/pure/onfido/onfido-sdk-ui/tree/master/demo/fiddle) for you to get you started.

**No idea**

---

**Comment by mountiny** [comment]
@twisterdotcom I have updated it, I can post it if you dont mind and we can close this issue until they would responds.

**Comment by mountiny** [comment]
Issue has been created [here](https://github.com/onfido/onfido-sdk-ui/issues/2139), I will close this issue out and try to update here if we get an update there!

**Comment by twisterdotcom** [comment]
Thanks!

**Comment by mountiny** [comment]
Following up here, the maintainers told us to update to latest package version so I created issue for that here https://github.com/Expensify/App/issues/14397

I am not sure yet if there are any breaking changes so that is the main thing we have to look into

**Comment by twisterdotcom** [comment]
We shouldn't bother updating the SDK. We're building on their API anyway.

**Comment by JmillsExpensify** [comment]
I think I agree with that comment, primarily because the API project is unblocked now. Otherwise, might not haha.
